Update planning-hub-sites.md (#3168)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/planning-hub-sites.md

|**Content authors** <br/> |**All members are content authors** who jointly create and edit content.  <br/> |**Small number of content authors** and a much larger number of content readers or consumers.  <br/> |**Hub site owner** defines the shared experiences for hub navigation and theme. **Hub site members** create content on the hub site as with any other SharePoint site. Owners and members of the sites associated with the parent hub create content on individual sites.  <br/> |
|**Governance** <br/> (as allowed for your organization based on the settings in the Security &amp; Compliance center)  <br/> |Norms typically **determined by the team**. Practices are aligned in the best way to get work done.  <br/> |Policies often **determined by the organization** to ensure consistency of experience and effective management of organizational information.  <br/> |Governance **determined by each owner of the associated site based on the type of site and organizational policies**. The best experience for visitors is achieved when everyone has at least read permissions for associated sites (but this is not required).  <br/> |

> |**Permissions** <br/> |Microsoft 365 group, plus SharePoint groups and permission levels  <br/> |SharePoint group  <br/> |Same as original site type. Hub sites do not alter an associated site's permissions - but you may add a "reader" group to the hub to make it easier to provide read access to associated sites. See: [Hub permissions](https://support.microsoft.com/en-us/office/set-up-your-sharepoint-hub-site-e2daed64-658c-4462-aeaf-7d1a92eba098#bkmk_managesiteassociationapprovals). <br/> |

Reviewer: ```suggestion
|**Permissions** <br/> |Microsoft 365 group, plus SharePoint groups and permission levels  <br/> |SharePoint group  <br/> |Same as original site type. Hub sites do not alter an associated site's permissions - but you may add a "reader" group to the hub to make it easier to provide read access to associated sites. For more information, see: [Hub permissions](https://support.microsoft.com/office/set-up-your-sharepoint-hub-site-e2daed64-658c-4462-aeaf-7d1a92eba098#bkmk_managesiteassociationapprovals). <br/> |
```

    
### Association

A site becomes part of a hub family by [Associating a SharePoint site with a hub site](https://support.office.com/article/ae0009fd-af04-4d3d-917d-88edb43efc05). When creating a hub site, SharePoint admins can [allow only certain site owners to associate sites with the hub](create-hub-site.md).
  
After a SharePoint admin gives a site owner permission to associate their sites with a hub site, the site owner can then choose to associate the sites with the hub. When they do, the site inherits the hub site theme and shared navigation. Content from their site will roll up to the hub site in web parts where the source is "all sites in the hub," and the site will be included in the hub site search scope.
  
Associating with the hub does not automatically add the site to the hub navigation. Hub site owners determine which sites are included in the navigation. They can also configure the News, Sites, Events, and Highlighted content to roll up activity from all associated sites or only selected sites.
  
> [!NOTE]

> > Association with a hub does not change the permissions on a site. If you associate a site that has restricted access with a hub, only users who have access to the restricted site will see content rolled up on the hub. Information surfaced on the hub site is security trimmed: if you don't have access to the content, you won't see it. Something you may want to consider is adjusting permissions on the associated sites after you have assembled your hub family or [adding a hub "read" permission gorup to the hub](https://support.microsoft.com/en-us/office/set-up-your-sharepoint-hub-site-e2daed64-658c-4462-aeaf-7d1a92eba098#bkmk_managesiteassociationapprovals) and adding that permission group to associated sites. 

Reviewer: ```suggestion
> Association with a hub does not change the permissions on a site. If you associate a site that has restricted access with a hub, only users who have access to the restricted site will see content rolled up on the hub. Information surfaced on the hub site is security trimmed: if you don't have access to the content, you won't see it. Something you may want to consider is adjusting permissions on the associated sites after you have assembled your hub family or [adding a hub "read" permission gorup to the hub](https://support.microsoft.com/office/set-up-your-sharepoint-hub-site-e2daed64-658c-4462-aeaf-7d1a92eba098#bkmk_managesiteassociationapprovals) and adding that permission group to associated sites. 
```

  
### Navigation

The hub site owner determines which sites are reflected in the shared navigation and can also include links to other resources. This navigation appears at the top, below the suite bar. Most of the time, you will want to add associated sites to your hub navigation. That's one of the benefits of the experiences that you can enable with a hub. Your hub navigation can have up to three levels, which lets you organize your hub family in a way that helps users discover and find relevant content.
  
However, you may not want to add every associated site to your navigation and you may want to consider adding sites that aren't associated to the navigation. Consider the following as you plan your hub navigation.
  

> - **Do you want to add private or restricted access sites to the navigation?** Maybe. For example, HR may want to associate their private team site with the HR hub to make it more convenient for HR team members. But, the HR hub owner may not want to display a link to the HR team site in the shared navigation for the HR hub because this would make the private HR site more discoverable by everyone in the organization, who will get an access challenge when they click the link to the HR team site. If you add private sites to hub navigation, consider using [audience targeting](https://support.microsoft.com/en-us/office/target-navigation-news-files-links-and-web-parts-to-specific-audiences-33d84cb6-14ed-4e53-a426-74c38ea32293) so that the link only appears for members of the private site. In another scenario, there may be a site that is "semi-private" that you want interested people to discover. For example, you might have a community that wants to restrict membership to people with a specific expertise, but also wants to discover experts across the organization. In this scenario, users might get an access denied/request access message, but the site owner is prepared and wants to grant access to interested people. 

Reviewer: ```suggestion
- **Do you want to add private or restricted access sites to the navigation?** Maybe. For example, HR may want to associate their private team site with the HR hub to make it more convenient for HR team members. But, the HR hub owner may not want to display a link to the HR team site in the shared navigation for the HR hub because this would make the private HR site more discoverable by everyone in the organization, who will get an access challenge when they click the link to the HR team site. If you add private sites to hub navigation, consider using [audience targeting](https://support.microsoft.com/office/target-navigation-news-files-links-and-web-parts-to-specific-audiences-33d84cb6-14ed-4e53-a426-74c38ea32293) so that the link only appears for members of the private site. In another scenario, there may be a site that is "semi-private" that you want interested people to discover. For example, you might have a community that wants to restrict membership to people with a specific expertise, but also wants to discover experts across the organization. In this scenario, users might get an access denied/request access message, but the site owner is prepared and wants to grant access to interested people. 
```

  
## Can I make just one hub site for my whole organization?

There is no requirement to have more than one hub for your organization, but you should think about what this means for information organization and discovery. An advantage to having just one hub is that every site in the intranet will share a consistent top navigation. However, since every site can also share consistent global navigaiton in the App Bar, you may want to consider taking advantage of more than one hub.
  

> If you have only one hub, you'll miss the ability to easily surface related information in context and the ability to easily define a search scope for related content. For example, if you have a single enterprise hub, it will be harder to surface just HR-related news on the HR site. Even small organizations may find that restricting the context in which users find information is helpful in managing information overload. We now have the ability to create a hierarchy of hubs to create [hub to hub associatons](https://docs.microsoft.com/en-us/sharepoint/hub-to-hub-association). This allows you to create a network of hubs that roll-up to each other to create connections and additional search scopes. When hubs are associated with each other, content can be searched for and displayed on hubs up to three levels of association.

Reviewer: ```suggestion
If you have only one hub, you'll miss the ability to easily surface related information in context and the ability to easily define a search scope for related content. For example, if you have a single enterprise hub, it will be harder to surface just HR-related news on the HR site. Even small organizations may find that restricting the context in which users find information is helpful in managing information overload. We now have the ability to create a hierarchy of hubs to create [hub to hub associatons](/sharepoint/hub-to-hub-association). This allows you to create a network of hubs that roll-up to each other to create connections and additional search scopes. When hubs are associated with each other, content can be searched for and displayed on hubs up to three levels of association.
```

  
## Additional important considerations

- **Finding hubs if you have more than one**. Hubs are an important building block for your intranet. Here are some ways you can make your hub sites discoverable: 
    

>   - **Add hubs to global navigation**. Add your hubs to the global navigation for your tenant in the [SharePoint app bar](https://docs.microsoft.com/en-us/viva/connections/sharepoint-app-bar). 

Reviewer: ```suggestion
  - **Add hubs to global navigation**. Add your hubs to the global navigation for your tenant in the [SharePoint app bar](/viva/connections/sharepoint-app-bar). 
```
